src/iuse.cpp

```diff
@@ -4667,17 +4667,47 @@ int iuse::oxytorch( player *p, item *it, bool, const tripoint & )
         return 0;
     }
 
-    const cata::optional<tripoint> pnt_ = choose_adjacent( _( "Cut up metal where?" ) );
-    if( !pnt_ ) {
-        return 0;
-    }
-    const tripoint pnt = *pnt_;
+    const static std::set<ter_id> allowed_ter_id {
```

Comment:
This is a static variable, initialized when it is first used, but the values stored in it depend on the data loaded from JSON (e.g. if "t_bars" is loaded before "t_reb_cage", it will have a different id than when it's loaded after it). This should store string ids (`ter_str_id`). Same for the furniture.

Reply:
Yes, looks like `static` is wrong here. Thanks!